examples/rosettacode/factorial.nit

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env nit
+#
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+
+# Task: Factorial
+# SEE: <http://rosettacode.org/wiki/Factorial>
+
+module factorial
+
+redef class Int
+
+    fun fact: Int
+    do
+        if self < 1 then return 1
+        return self * (self-1).fact
+    end
+end
+var n = gets.to_i
+print n.fact
```

Comment:
I just see that there is a missing line-feed at the end of the file, git likes to complain about not having them. What text editor did you use already?